### PR TITLE
Remove myself from code owners

### PR DESCRIPTION
**Pull request by asarium**
I have not been able to keep up with code reviews for quite some time so I should not be a default one.

### CODE REVIEW

File: cmake/util.cmake [hunk 1]
```diff
@@ -70,7 +70,7 @@ MACRO(COPY_FILE_TO_TARGET _target _file)
 	if(UNIX)
 		ADD_CUSTOM_COMMAND(
 			TARGET ${_target} POST_BUILD
-			COMMAND cp -a "${_file}"  "$<TARGET_FILE_DIR:${_target}>/${LIBRAY_DESTINATION}/"
+			COMMAND cp -au "${_file}"  "$<TARGET_FILE_DIR:${_target}>/${LIBRAY_DESTINATION}/"
```
Comment (The-E) [inline cmake/util.cmake:73]: this seems like an accidental change? It breaks compilation on MacOS....

**Review by The-E — CHANGES_REQUESTED** [review]

File: cmake/util.cmake (hunk 1 above)
Comment (asarium) [reply cmake/util.cmake]: There was a build break in a previous iteration where a file was copied twice. I looked up the appropriate option in my `man cp` but of course Mac has to be special with their tools. I changed it to always use the CMake variant.

Comment (asarium) [reply cmake/util.cmake]: Whenever I change something, another thing breaks, I'll just revert these changes...

**Review by The-E — APPROVED** [review]

**Comment by asarium** [comment]
Seems like the error is back but since no code changes were made, I will just force the merge.

**Pull request merged**